browser/src/control/Control.UIManager.js

```diff
@@ -360,8 +360,14 @@ L.Control.UIManager = L.Control.extend({
 
 		this.map.on('changeuimode', this.onChangeUIMode, this);
 
-		if (typeof window.initializedUI === 'function')
+		if (typeof window.initializedUI === 'function') {
 			window.initializedUI();
+		}
+
+		var startPresentationGet = this.map.isPresentationOrDrawing() && window.coolParams.get('startPresentation');
+		if (startPresentationGet == true || startPresentationGet === 'true') {
```

Comment:
```suggestion
		if (startPresentationGet === true || startPresentationGet === 'true') {
```

Reply:
I used '==' on purpose to be more input tolerant, i.e accepting 1 as true.